src/Directives/DefaultDirective.php

```diff
@@ -0,0 +1,148 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Mazedlx\FeaturePolicy\Directives;
+
+use Mazedlx\FeaturePolicy\Directive;
+use Mazedlx\FeaturePolicy\DirectiveContract;
+use Mazedlx\FeaturePolicy\Exceptions\InvalidDirective;
+
+final class DefaultDirective
+{
+    public static function directive(string $directive): DirectiveContract
+    {
+        return match ($directive) {
```

Comment:
using anonymous classes, because creating separate class files for only implementing the `name()` method seemed overkill.